packages/ui/components/combobox/src/LionCombobox.js

```diff
@@ -642,14 +657,27 @@ export class LionCombobox extends OverlayMixin(LionListbox) {
    * @param {string} matchingString
    * @protected
    */
-  // eslint-disable-next-line class-methods-use-this
   _onFilterMatch(option, matchingString) {
-    makeMatchingTextBold(option, matchingString, option);
+    this._highlightMatchedOptions(option, matchingString);
```

Comment:
It would be nice if these would be tested as well in Subclasser describe and its use case would be clear from it.

Since these methods only handle one option a time, maybe we could consider renaming to:
- `_highlightMatchedOption`
- `_unhighlightMatchedOption`